package.json

```diff
@@ -95,6 +95,6 @@
     "qs": "^6.5.1"
   },
   "peerDependencies": {
-    "algoliasearch": ">= 3.1 < 4"
+    "algoliasearch": ">= 4.0"
```

Comment:
changing this would be a major version, doesn't the API still accept both v3 & v4 in compat mode??

Reply:
Let's discuss this tomorrow in our meeting.